apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -502,6 +513,40 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     {chain_validity, events}
   end
 
+  @spec get_exit_info(Utxo.Position.t(), t()) :: {:ok, ExitInfo.t()} | {:ok, :not_found}
+  def get_exit_info(Utxo.position(_, _, _) = utxo_exit, %__MODULE__{exits: exits}) do
```

Comment:
mabe ther shoud be guard is_position

Reply:
I think it is enough to have:  `Utxo.position(_, _, _)`. why do you think the guard is needed?